js/web/lib/onnxjs/backends/webgl/ops/conv-pack.ts

```diff
@@ -16,6 +16,7 @@ import {WebGLReshapePacked} from './reshape-packed';
 export class WebGLConvPacked extends Conv {
   protected artifacts: Artifact[];
   protected programInfo: ProgramInfo[];
+  protected outputShape: number[];
```

Comment:
why do we need to add this as class field?

Reply:
we cache the outputshape for each occurrence of this op to save on the outputshape computation for each run.